src/standalone/Bucket/Bucket.jl

```diff
@@ -62,86 +61,63 @@ include(
+An albedo model where the static snow-free bareground albedo is prescribed as a
+function of space or using data from a file, and the land surface albedo is
+computed each timestep as a linear combination of the snow albedo and the bareground
+albedo, following the SLIM model (Lague et al 2019).
 """
-struct BulkAlbedoFunction{FT, F <: ClimaCore.Fields.Field} <:
-       AbstractLandAlbedoModel{FT}
+struct PrescribedBaregroundAlbedo{FT, F <: ClimaCore.Fields.Field} <:
+       AbstractBucketAlbedoModel{FT}
     α_snow::FT
     α_bareground::F
 end
 
-function BulkAlbedoFunction{FT}(
-    α_snow::FT,
-    α_bareground::F,
-) where {FT <: AbstractFloat, F}
-    BulkAlbedoFunction(α_snow, α_bareground)
-end
-
-function BulkAlbedoFunction{FT}(
-    α_snow::FT,
-    α_bareground_func::Function,
-    space,
-) where {FT <: AbstractFloat}
-    α_bareground = SpaceVaryingInput(α_bareground_func, space)
-    args = (α_snow, α_bareground)
-    BulkAlbedoFunction{typeof.(args)...}(args...)
-end
-
 """
-    BulkAlbedoStatic{FT, F <: ClimaCore.Fields.Field} <: AbstractLandAlbedoModel
+     PrescribedBaregroundAlbedo{FT}(α_snow::FT,
+                                    α_bareground_func::Function,
+                                    surface_space::ClimaCore.Spaces.AbstractSpace
+                                    ) where {FT}
 
-An albedo model where the albedo of different surface types
-is specified. Snow albedo is treated as constant across snow
-location and across wavelength. Bareground surface albedo is specified
-via a NetCDF file that is treated
-as constant across wavelengths; surface is this context refers
-to soil and vegetation. This albedo type is static in time.
+An outer constructor for the PrescribedBaregroundAlbedo model which uses an analytic function
+of the coordinates to compute α_bareground on the model `surface_space`.
 
-Note that this option should only be used with global simulations,
-i.e. with a `ClimaLand.LSMSphericalShellDomain.`
+This particular method can be used with site level or global runs.
 """
-struct BulkAlbedoStatic{FT, F <: ClimaCore.Fields.Field} <:
-       AbstractLandAlbedoModel{FT}
-    α_snow::FT
-    α_bareground::F
+function PrescribedBaregroundAlbedo{FT}(
+    α_snow::FT,
+    α_bareground_func::Function,
+    surface_space::ClimaCore.Spaces.AbstractSpace,
+) where {FT}
+    α_bareground = SpaceVaryingInput(α_bareground_func, surface_space)
+    return PrescribedBaregroundAlbedo{FT, typeof(α_bareground)}(
+        α_snow,
+        α_bareground,
+    )
 end
 
-function BulkAlbedoStatic{FT}(α_snow::FT, α_bareground::F) where {FT, F}
-    return BulkAlbedoStatic(α_snow, α_bareground)
-end
 
 """
-    BulkAlbedoStatic{FT}(
-        regrid_dirpath::String,
-        surface_space::ClimaCore.Spaces.AbstractSpace;
-        α_snow = FT(0.8),
-        varname = ["sw_alb"],
-        get_infile::Function = Bucket.bareground_albedo_dataset_path,
-    ) where {FT}
+     PrescribedBaregroundAlbedo{FT}(α_snow::FT,
```

Comment:
I also removed the default for alpha snow for consistency between the methods for this struct.